_posts/2021-04-09-three-ways-we-supported-the-past-signalen-community-meetup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Three ways we supported the past Signalen Community Meetup"
+date: 2021-04-09
+author: Alba Roza
+type: blogpost
+excerpt: Organizing a virtual event with the Signalen community
+categories:
+  - Signalen
+---
+
+# Three ways we supported the past Signalen Community Meetup
+
+Last March 22 one of the codebases we are working with, the Dutch [Signalen](signalen.org), organized its first Community Meetup. The aim of this virtual get together was to get to know new municipalities curious about the solution for nuisance reporting as well as vendors interested in its implementation.
```

Comment:
@AlbaRoza add https:// in link so`[Signalen](https://signalen.org)`

Reply:
Thanks!